docs/docs/docs/non-automatic-derivation.md

```diff
@@ -75,6 +75,39 @@ We are now ready to read `Person` configs:
 ConfigSource.fromConfig(conf).load[Person]
 ```
 
+### Semi-Automatic for Sealed Families
```

Comment:
nit: can you either remove this title or make it an h4? It looks like a subsection of "Semi-Automatic", not on the same level as "Automatic" and "Manual".

Reply:
@ruippeixotog Thanks for catching that. I meant to make it an h4 and simply missed.